frontends/web/src/i18n/en.json

```diff
@@ -648,6 +648,12 @@
                 "title": "How to securely pair with your phone",
                 "text": [ "After having downloaded our mobile app for either iOS or Android, you scan the displayed QR code, which sets up a secure channel between the mobile app and this application. Once scanned, follow the instructions in the mobile app." ]
             },
+            {   "title": "How does 2FA work",
+                "text": [
+                    "When 2FA is enabled, possession of a mobile phone is required in order to spend coins. Under the hood, an encrypted single-use PIN is sent to the mobile app, decrypted there, and returned to the BitBox when pressing the Accept button. This communication with the device is done via the channel between a mobile phone and this desktop app established during pairing.",
```

Comment:
Suggestions:
– "possession of a mobile phone is required in order to spend coins" –> "all transactions have to approved on the paired mobile phone" (I don't like the "a mobile phone" because it's a very specific one.)
– "PIN" –> "number" (I have no idea why Douglas called this PIN (also in the code). It has nothing to do with a personal identification number. Nonce and challenge would both be more suitable.)
– "between a mobile phone" –> "between the mobile phone"

Reply:
All done. Thanks!